docs/api/protocol-ns.md

```diff
@@ -0,0 +1,309 @@
+# protocolNS
+
+This document describes the new protocol APIs based on NetworkService.
+
+Currently we don't have an estimate of when will we enable NetworkService by
+default in Electron, but as Chromium is already removing non-NetworkService
+code, the switch may come after a few major version releases.
```

Comment:
```suggestion
code, we might switch before Electron 10.
```

See above: I'm not sure if that's the right number, but we should use something we don't have to update.